content/blog/2019-05-21-may-19-dvc-heartbeat.md

```diff
@@ -256,7 +256,7 @@ $ dvc metrics show metrics.json \
 
 There are a few options to add a new dependency:
 
-- simply opening a file with your favorite editor and adding a dependency there
+- opening a file with your favorite editor and adding a dependency there
```

Comment:
I don't think this instance of "simply" counts as condescending. It just means that the operation is not complex. Opening a file is quite simple. Or am I wrong based on whatever source you used to decide these, @kurianbenoy? Please share if so. I could def. be wrong.

Reply:
@jorgeorpinel isn't simply just an adverb in this case, won't it be better to remove this in this context?

For someone who may not be a programmer, maybe opening a file is not quite simple after all. I personally got that impression after [reading this article](https://dev.to/meeshkan/how-to-remove-condescending-language-from-documentation-4a5p#use-alex)

Reply:
Hey @kurianbenoy sorry for the delay, we've remained too busy with th 1.0 release but I think this week I can get back to GSoD work.

Reply:
OK you may be right but I didn't really think we would change anything in the blogs, just the docs section of our website (https://dvc.org/doc)

Let's leave it for now but let @andronovhopf decide on this one. Can you chime in please Elle? Thanks